applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -681,20 +681,9 @@ def ModifyProperties(self, properties, param = 0):
         else:
             translational_scheme_name = self.DEM_parameters["TranslationalIntegrationScheme"].GetString()
 
-        if properties.Has(PARTICLE_FRICTION):
-            self.Procedures.KratosPrintWarning("---------------------------------------------------")
-            self.Procedures.KratosPrintWarning("  WARNING: Property PARTICLE_FRICTION is deprecated ")
-            self.Procedures.KratosPrintWarning("  since April 11th, 2018, replace with FRICTION")
-            self.Procedures.KratosPrintWarning("  Automatic replacement is done now.")
-            self.Procedures.KratosPrintWarning("---------------------------------------------------")
-            properties[FRICTION] = properties[PARTICLE_FRICTION]
-        if properties.Has(WALL_FRICTION):
-            self.Procedures.KratosPrintWarning("-------------------------------------------------")
-            self.Procedures.KratosPrintWarning("  WARNING: Property WALL_FRICTION is deprecated")
-            self.Procedures.KratosPrintWarning("  since April 11th, 2018, replace with FRICTION")
-            self.Procedures.KratosPrintWarning("  Automatic replacement is done now.")
-            self.Procedures.KratosPrintWarning("-------------------------------------------------")
-            properties[FRICTION] = properties[WALL_FRICTION]
+        if properties.Has(FRICTION):
+            properties[STATIC_FRICTION] = properties[FRICTION]
```

Comment:
If I understood coorectly, we can print a Warning here, saying that FRICTION is no longer valid, right?

Reply:
Are we going to use these different frictions for the walls, too?

Reply:
In a first attempt, I wrote a Warning saying that FRICTION is no longer valid, however, I thought that it could be interesting if the user can set the FRICTION (only one parameter that automatically sets the other two) or the STATIC_FRICTION and DYNAMIC_FRICTION (two parameters). What do you think?

About the walls, I think it would be interesting to use different frictions also, this PR comes from the DEM benchmark for the angle of repose and, in that case, it was necessary to apply different static and dynamic friction to the walls.

Reply:
Nothing serious, but in my opinion there should not be several ways to do the same thing. At least for setting the friction. Right now we must leave the old way functional, for old cases and the GUI, but I would remove it eventually. No need to add it to this PR, if there is rush.

Reply:
I agree. Now, a Warning is printed when using FRICTION. I also update the tests to use STATIC_FRICTION and DYNAMIC_FRICTION.
All tests run correctly, tests_example 40 is failing but is not due to this PR, it is failing also in the master.